config/default/manager_auth_proxy_patch.yaml

```diff
@@ -32,3 +32,5 @@ spec:
         - "--health-probe-bind-address=:8081"
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
+        - "--image=$(RELATED_IMAGE_CONTROLLER)"
+        - "--namespace=$(TARGET_NAMESPACE)"
```

Comment:
Why these flags are in the auth proxy patch and not in `config/manager/manager.yaml`?

Reply:
Because that get's overriden and I haven't found a way to merge them.

Reply:
You can put all the flags of `manager` container in `config/manager/manager.yaml` including the ones from this PR:
```yaml
        args:
        - "--health-probe-bind-address=:8081"
        - "--metrics-bind-address=127.0.0.1:8080"
        - "--leader-elect"
        - "--image=$(RELATED_IMAGE_CONTROLLER)"
        - "--namespace=$(TARGET_NAMESPACE)"
```
Because the ones which are currently in the auth proxy patch should be in the manager regardless the fact if there is a proxy in front or not (leader election, health probe address). Even the metrics address makes sense without the proxy.

Reply:
Was done in #45